VISION.md

```diff
@@ -7,8 +7,9 @@
 
 # Scope
 
-**TBD**
+- We plan on providing Android support in [version 3.0](https://github.com/xing/calabash-launcher/milestone/3).
+- We **do not** plan on adding support to actually write Calabash tests.
```

Comment:
Interesting that we wanted to work on Calabash writer with Daniel, but definitely not as a part of launcher. Could we add ... as a part of the given project. ?